### PR TITLE
Show correct message when invalid arguments are passed to brew

### PR DESCRIPTION
**Pull request by jpninanjohn**
-----

Explanation:

Closes [#69937](https://github.com/Homebrew/homebrew-cask/issues/69937)
I have added a block to check specifically for help arguments. If it is not the help command then it will fail saying brew cask did not recognize the command. 

Help needed:

For running the tests/style checks, I made changes directly to my homebrew code and ran `brew style` and `brew tests`. Not sure if this is the correct way to do it. Please correct if there is a betetr way of testing or if I have missed anything. 

@unitof Tagging you here since you were also working on this.

### CODE REVIEW

File: Library/Homebrew/cask/cmd.rb [hunk 1]
```diff
@@ -221,9 +221,14 @@ def run(*_args)
...
-        return if @command == "help" && @args.empty?
 
-        raise ArgumentError, "help does not take arguments."
+        if @command == "help"
+          return if @args.empty?
+
+          raise ArgumentError, "help does not take arguments." if @args.length
+        end
+
+        raise ArgumentError, "brew cask does not recognise this command"
```
Comment (unitof) [inline Library/Homebrew/cask/cmd.rb]: For consistent with `brew`'s invalid command error messages, I think this should be:
```
raise ArgumentError, "Unknown Cask command: #{command}"
```

If you want to give me write access to your repo I'll happily commit that.

Comment (jpninanjohn) [reply Library/Homebrew/cask/cmd.rb]: Will do. Go ahead and make the change. Thanks

Comment (jpninanjohn) [reply Library/Homebrew/cask/cmd.rb]: Have added you as collab -> https://github.com/jpninanjohn/brew/invitations

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @jpninanjohn!

**Pull request merged**